HotelWebsite/contact.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Welcome To Contact</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div class="container">
+                <h1 class="logo">HBT</h1>
+                <ul>
+                    <li><a href="index.html"> HOME</a></li>
+                    <li><a href="about.html">ABOUT </a></li>
+                    <li><a class="current" href="contact.html">CONTACT</a></li>
+                </ul>
+            </div>
+        </nav>
+
+    </header>
+    <section id="contact-form" class="py3">
+        <div class="container">
+            <h1 class="l-heading"> Contact <span class="primary-heading"> Us</span></h1>
+            <p>Please fill out the form below to contact us</p>
+            <form action="process.php">
+                <div class="form-group">
```

Comment:
can use `__` for naming in case of classes, it is also selected at once when you press Control D on VSCODE.

For example,
`form__input__field`